src/notebook/notebookKernel.ts

```diff
@@ -88,9 +87,6 @@ export class JuliaKernel {
                     const runStartTime = Date.now()
                     this._currentExecutionRequest.start(runStartTime)
 
-                    // TODO Ideally we would clear output at scheduling already, but for now do it here
-                    await this._currentExecutionRequest.clearOutput()
```

Comment:
Clearing of output via the Execution task can only be done after you start the execution task.
So if we want to clear output when we queue, then we'll need to create an execution task as follows:
* Create execution task
* Call start on the execution task
* Clear the output
* End the execution task 

I.e. we must call start before calling clear.
In this case its best to create a whole new execution task prior to queueing of the cells.